presto-hive/src/main/java/io/prestosql/plugin/hive/HiveConnector.java

```diff
@@ -28,6 +28,7 @@
 import io.prestosql.spi.connector.ConnectorSplitManager;
```

Comment:
I would simply write `Allow HiveConnector to return EventListener` for commit message.

Terms like `external event listeners` or `hive plugin ecosystem` are a bit misleading. Maybe this just a matter of taste.

Reply:
sure